examples/jupyter_lite_config.json

```diff
@@ -3,7 +3,7 @@
     "contents": ["."],
     "federated_extensions": [
       "https://github.com/conda-forge/releases/releases/download/noarch/bqplot-0.12.30-pyhd8ed1ab_0.tar.bz2/bqplot-0.12.30-pyhd8ed1ab_0.tar.bz2",
-      "https://github.com/conda-forge/releases/releases/download/noarch/ipycanvas-0.9.1-pyhd8ed1ab_0.tar.bz2/ipycanvas-0.9.1-pyhd8ed1ab_0.tar.bz2",
```

Comment:
Was there a reason for downgrading ipycanvas?

IIRC we need ipycanvas 0.9.1 specifically for the jupyterlite lua kernel. We can always pin ipycanvas correctly in the example deployments of lua though.

Reply:
no particular reason, probably a bad merge or something... this PR's getting a little long in the tooth.

Reply:
`0.9.1` is back live, again... we can now actually _drop_ all the in-notebook version pins, if we like, as `piplite` will take care of getting the correct version for us... but we can also hold off on that for a separate PR.